docs_user/modules/proc_configuring-control-plane-services.adoc

```diff
@@ -0,0 +1,106 @@
+[id="configuring-control-plane-services_{context}"]
+
```

Comment:
This is one thing that i've found strange on the downstream docs filesystem structure. The heading "Configuring control plane services" makes sense in the rendered document because it is a subheading under "Network requirements and configuration" so it's natural that we're talking about configuring control plane services in the broader context of network configuration. But the file name is "proc_configuring-control-plane-services.adoc" which may lead one to think that it is actually about configuring the control services themselves, rather than configuring the networking to make it ready for the control plane services. Configuration of the services themselves is handled elsewhere in the documentation, so that makes the file name somewhat misleading, to me at least.

So this is not a -1, if that's how we've been doing things (and from other docs i've seen it looks like that), then let's not break the pattern. So maybe this is just feedback for longer term consideration within the docs team. (I'm wondering whether you folks can navigate this comfortably :) .)

Reply:
In this case, you're right that we should be more specific, especially since I'm going to create a chapter called "Adopting OpenStack control plane services" in another PR. The similar names might cause confusion. Maybe this instead?
Title of module: Configuring the networking for control plane services
Filename: proc_configuring-the-networking-for-control-plane-services.adoc

Reply:
+1